tests/learn-ocaml-tests.el

```diff
@@ -63,6 +63,7 @@
 (defun learn-ocaml-test-collapse-whitespace (str)
   (replace-regexp-in-string "[[:space:]\n]+" " " str))
 
+
```

Comment:
Maybe this extra newline is unneeded; can you revert it?

Reply:
- Yep, it's done for the extra newline.
- I have reverted changes concerning token's update in order to do it in another Pull Request. I didn't meant to do it here, that was a mistake. I think it's gonna be better like this.
- I have also squash all commits, but I have a little question. You can squash commits in a pull request right before merging, right?

Reply:
@LouisAyroles
> I have a little question. You can squash commits in a pull request right before merging, right?

Indeed, that's a working feature of GitHub (Merge-and-Squash a PR) that I use sometimes,
but in this case it wouldn't have been applicable with my first suggestion (for keeping *two* different commits, not one).

Reply:
Oh ok, i understand, thank you :)